fn-fdk.js

```diff
@@ -223,6 +223,20 @@ function getInputHandler (inputMode) {
   }
 }
 
+function logFramer (ctx) {
+  let framer = process.env['FN_LOGFRAME_NAME']
+  if (framer !== '') {
+    let valueSrc = process.env['FN_LOGFRAME_HDR']
+    if (valueSrc !== '') {
+      let id = ctx.getHeader(valueSrc)
+      if (id !== '') {
+        console.log('\n' + framer + '=' + id + '\n')
+        console.error('\n' + framer + '=' + id + '\n')
```

Comment:
Both console.log and console.error already append a newline, so trailing newline is not needed I think in this case.